lib/streamlit/config.py

```diff
@@ -438,6 +438,16 @@ def _server_run_on_save():
     return False
 
 
+@_create_option("server.address")
+def _server_address():
+    """The Address where the server will listen for client and browser
+    connections.
```

Comment:
Please change to this:

```
The address where the server will listen for client and browser connections. Use this if you want to bind the server to a specific address. If set, the server will only be accessible from this address, and not from any aliases (like localhost).
```

Also, while you're at it, can you change the doc for `server.port` to this please?

```
The port where the server will listen for browser connections.
```

😉

Reply:
Fixed it.